catalog/source/workflows.yml

```
@@ -685,6 +685,194 @@ workflows:
+        type_guide:
+          class: Collection
+          ext: bam
+      - key: Depth-threshold for masking
+        type_guide:
+          class: integer
+      - key: Reference genome
+        type_guide:
+          class: File
+    active: false
+  - trs_id: "#workflow/github.com/iwc-workflows/sars-cov-2-ont-artic-variant-calling/COVID-19-ARTIC-ONT/versions/v0.3.2"
+    categories:
+      - VARIANT_CALLING
+    workflow_name: "COVID-19: variation analysis of ARTIC ONT data"
+    workflow_description:
+      This workflow for ONT-sequenced ARTIC data is modeled after
+      the alignment/variant-calling steps of the [ARTIC pipeline](https://artic.readthedocs.io/en/latest/).
+      It performs, essentially, the same steps as that pipeline’s minion command, i.e.
+      read mapping with minimap2 and variant calling with medaka. Like the Illumina
+      ARTIC workflow it uses ivar for primer trimming. Since ONT-sequenced reads have
+      a much higher error rate than Illumina-sequenced reads and are therefor plagued
+      more by false-positive variant calls, this workflow does make no attempt to handle
+      amplicons affected by potential primer-binding site mutations.
+    ploidy: ANY
+    parameters:
+      - key: ONT-sequenced reads
+        type_guide:
+          class: Collection
+          ext:
+            - fastqsanger
+            - fastqsanger.gz
+      - key: Minimum read length
+        type_guide:
+          class: integer
+      - key: Maximum read length
+        type_guide:
+          class: integer
+      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
+        type_guide:
+          class: File
+          ext:
+            - fasta
+            - fasta.gz
+      - key: Primer binding sites info in BED format
+        type_guide:
+          class: File
+          ext: bed
+    active: false
+  - trs_id: "#workflow/github.com/iwc-workflows/sars-cov-2-pe-illumina-artic-variant-calling/COVID-19-PE-ARTIC-ILLUMINA/versions/v0.5.3"
+    categories:
+      - VARIANT_CALLING
+    workflow_name: "COVID-19: variation analysis on ARTIC PE data"
+    workflow_description:
+      The workflow for Illumina-sequenced ARTIC data builds on the
+      RNASeq workflow for paired-end data using the same steps for mapping and variant
+      calling, but adds extra logic for trimming ARTIC primer sequences off reads with
+      the ivar package. In addition, this workflow uses ivar also to identify amplicons
+      affected by ARTIC primer-binding site mutations and tries to exclude reads derived
+      from such tainted amplicons when calculating allele-frequencies of other variants.
+    ploidy: HAPLOID
+    taxonomy_id: 694009
+    parameters:
+      - key: Paired Collection
+        type_guide:
+          class: Collection
+          ext:
+            - fastqsanger
+            - fastqsanger.gz
+      - key: NC_045512.2 FASTA sequence of SARS-CoV-2
+        variable: ASSEMBLY_ID
+      - key: ARTIC primer BED
```

I think there are hardcoded urls to this bed file ? @hunterckx it would be good if we can directly specify parameters here.
Ideally we'd write:
```suggestion
      - key: ARTIC primer BED
         value:
           ext: bed
           src: url
           url: https://zenodo.org/record/4555735/files/ARTIC_nCoV-2019_v3.bed
```